tests/test_http.py

```diff
@@ -4,7 +4,7 @@
 import signal
 import threading
 import time
-from func_python.http import serve
+from func_http.http import serve
```

Comment:
With an `http.py` and a `cloudevents.py` file in the `func_python` directory, the two imports would be:
`from func_python.http import serve` for HTTP  middleware and
`from func_python.cloudevents import serve` for CloudEVent middleware
That seems correct to me